modules/metaphlan3/mergemetaphlantables/main.nf

```diff
@@ -0,0 +1,33 @@
+process METAPHLAN3_MERGEMETAPHLANTABLES {
+    label 'process_low'
+
+    conda (params.enable_conda ? 'bioconda::metaphlan=3.0.12' : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metaphlan:3.0.12--pyhb7b1952_0' :
+        'quay.io/biocontainers/metaphlan:3.0.12--pyhb7b1952_0' }"
+
+    input:
+    path(profiles)
+
+    output:
+    tuple path("${prefix}.txt") , emit: txt
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "merged_abundance_table"
+    """
+    merge_metaphlan_tables.py \\
+        $args \\
+        -o ${prefix}.txt \\
```

Comment:
if no multithreading, would the `process_single` label be enough?

Reply:
Yes! I keep forgetting about this, I just run through the default on the dialogue :facepalm: